src/context.rs

```diff
@@ -179,7 +179,8 @@ pub fn dc_context_new(
         ))),
         probe_imap_network: Arc::new(RwLock::new(0)),
         perform_inbox_jobs_needed: Arc::new(RwLock::new(0)),
-    }
+    };
+    context
```

Comment:
i find it more readable to have short things be used as expression-returns.

Reply:
but very non rust style, please dont :)

Reply:
fixed